crates/bevy_render/src/mesh/shape/icosphere.rs

```diff
@@ -64,13 +64,13 @@ impl From<Icosphere> for Mesh {
             );
         }
         let generated = IcoSphere::new(sphere.subdivisions, |point| {
-            let inclination = point.z.acos();
-            let azumith = point.y.atan2(point.x);
+            let inclination = point.y.acos();
+            let azimuth = point.z.atan2(point.x);
 
-            let norm_inclination = 1.0 - (inclination / std::f32::consts::PI);
-            let norm_azumith = (azumith / std::f32::consts::PI) * 0.5;
+            let norm_inclination = inclination / std::f32::consts::PI;
+            let norm_azimuth = (azimuth / std::f32::consts::TAU) + 0.5;
```

Comment:
shouldn't this be `1.0 - ((azimuth / std::f32::consts::TAU) + 0.5);` ? otherwise the texture is inverted on the x axis

Reply:
yeah africa is inverted. the tweak with the subtraction fixes it
![CleanShot 2021-04-10 at 14 14 30](https://user-images.githubusercontent.com/11653/114284907-11900880-9a08-11eb-9e70-cc37465e8fb1.png)

Reply:
Ah, good catch, will update to reflect this.